system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -24,45 +59,68 @@
                 	</div>
                 </div>
         			</div>
+
+              <script>
+                function searchFilter() {
+                var input, filter, table, tr, td, i, txtValue;
+                input = document.getElementById("Search_Term");
+                filter = input.value.toUpperCase();
+                table = document.getElementById("main_Table");
+                tr = table.getElementsByTagName("tr");
+                for (i = 0; i < tr.length; i++) {
+                  td = tr[i].getElementsByTagName("strong")[0];
+                  if (td) {
+                    txtValue = td.textContent || td.innerText;
+                    if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                      tr[i].style.display = "";
+                    } else {
+                      tr[i].style.display = "none";
+                    }
+                  }       
+                }
+              }
+
+              function typeFilter(input){
+                var table, vr;
+                table = document.getElementById("main_Table");
+                tr = table.getElementsByTagName("tr");
+
+                for (i = 0; i < tr.length; i++) {
+                   if(tr[i].className == input){
+                    tr[i].style.display = "";
+                   }else {
+                      tr[i].style.display = "none";
+                   }
+                }
+                return false;
+              }
+              </script>
+
+
+
     					<div class="filter-bar__item ">
-        				<div class="filter-search-bar__item">
-                	<button type="button" class="has-sub filter-bar__button js-dropdown-toggle button button--default button--small" data-filter-label="show">show				<span class="faded">(25)</span>
-            			</button>
-                	<div class="dropdown">
-            				<div class="dropdown__search">
-                			<div class="search-input">
-                  			<input type="text" name="perpage" value="" placeholder="custom limit" data-threshold="1000" data-threshold-text="Viewing more than 1000 items at a time may result in reduced performance." class="search-input__input input--small">
-                			</div>
-                		</div>
-										<a class="dropdown__link" href="">25 results</a>
-										<a class="dropdown__link" href="">50 results</a>
-										<a class="dropdown__link" href="">75 results</a>
-										<a class="dropdown__link" href="">100 results</a>
-										<a class="dropdown__link" href="">150 results</a>
-										<a class="dropdown__link" href="">All Add-Ons</a>
-      						</div>
-                </div>
+
         			</div>
           	</div>
           </div>
         </div>
       </div>
       <div class="table-responsive table-responsive--collapsible">
-        <table cellspacing="0">
+        <table id = "main_Table" cellspacing="0">
```

Comment:
I believe we should not start usin spaces around `=`, as those are currently not used. Let's stick to same coding style